pkg/ruler/ruler.go

```diff
@@ -50,6 +50,7 @@ const (
 	ShardingStrategyDefault = "default"
 	ShardingStrategyShuffle = "shuffle-sharding"
 
+	// Number of concurrent group list and group loads operations.
 	loadRulesConcurrency = 10
```

Comment:
Might make sense to move this variable to `pkg/ruler/storage.go` since that is where it is used.

Reply:
It is also used in the ruler.go in sharded load.